pom.xml

```diff
@@ -15,7 +15,7 @@
   <parent>
     <groupId>org.sonatype.nexus.plugins</groupId>
     <artifactId>nexus-plugins</artifactId>
-    <version>3.23.0-03</version>
+    <version>3.24.0-02</version>
```

Comment:
I would suggest to develop it on top of the 3.25.0-SNAPSHOT

Reply:
did it 3.26.0-SNAPSHOT